src/main/java/org/folio/dew/batch/bursarfeesfines/service/impl/BursarExportServiceImpl.java

```diff
@@ -80,13 +86,59 @@ private BursarExportJob extractBursarFeeFines(JobParameter jobParameter)
   /** Take the found fine accounts and mark them as transferred */
   @Override
   public void transferAccounts(
-    List<Account> accounts,
+    List<AccountWithAncillaryData> accounts,
     BursarExportJob bursarFeeFines
   ) {
-    log.error("I don't know how to create transfer requests yet...");
-    // var transferRequest = toTransferRequest(accounts, bursarFeeFines);
-    // log.info("Creating {}.", transferRequest);
-    // bulkClient.transferAccount(transferRequest);
+    Set<AccountWithAncillaryData> transferredAccountsSet = new HashSet<>();
+    Set<AccountWithAncillaryData> nonTransferredAccountsSet = new HashSet<>(
+      accounts
+    );
+
+    for (BursarExportTransferCriteriaConditionsInner bursarExportTransferCriteriaConditionsInner : bursarFeeFines
+      .getTransferInfo()
+      .getConditions()) {
+      List<AccountWithAncillaryData> accountsToBeTransferred = accounts
+        .stream()
+        .filter(account ->
+          BursarFilterEvaluator.evaluate(
+            account,
+            bursarExportTransferCriteriaConditionsInner.getCondition()
+          )
+        )
+        .collect(Collectors.toList());
+      transferredAccountsSet.addAll(accountsToBeTransferred);
+
+      String accountName = getTransferAccountName(
+        bursarExportTransferCriteriaConditionsInner.getAccount().toString()
+      );
+
+      log.info(
```

Comment:
Please check that there are > 0 accounts for each condition before making transferRequests